app/org/sagebionetworks/bridge/dynamodb/DynamoUploadDao.java

```diff
@@ -93,6 +100,10 @@ public void writeValidationStatus(@Nonnull Upload upload, @Nonnull UploadStatus
         upload2.setRecordId(recordId);
 
         // persist
-        mapper.save(upload2);
+        try {
+            mapper.save(upload2);
+        } catch (ConditionalCheckFailedException ex) {
+            throw new ConcurrentModificationException("Unable to write validation status for upload " + upload.getUploadId());
+        }
```

Comment:
Also, this change is probably unnecessary, since this is only being called by the async upload validation pipeline. This means that the error is never bubbled up to the service response, so there's no difference between a ConditionalCheckFailedException and a ConcurrentModificationException.

If we do keep this change, we want to preserve the original error message and stack trace as well.